tests/test_gpu_common.py

```diff
@@ -1442,6 +1442,18 @@ def test_pickling(self):
 
         assert str(op) == str(new_op)
 
+    def test_tensor_fit(self):
```

Comment:
There's a TestAPI class in this file I'd move it there and spell it out `test_gpu_fit_w_tensor_functions`

(Spelling it out helps in absence of docstrings)